android/gltfio-android/gradle.properties

```diff
@@ -1,3 +1,4 @@
 POM_NAME=Filament glTF Loader
-POM_ARTIFACT_ID=gltfio-android
+POM_ARTIFACT_ID_FULL=gltfio-android-full
```

Comment:
This changing the name of the Maven artifact. Can you update the root README?

Reply:
After thinking about this, I think it's better to keep the old name.  It's less disruptive to current users (like Tim) and the full version is still < 1 MB so IMO it should still considered the "default" flavor.  The root README now mentions the lite flavor.